geotrek/flatpages/static/flatpages/tinymce/js/additional_tinymce_plugins.js

```diff
@@ -80,32 +80,32 @@ document.addEventListener("DOMContentLoaded", function () {
                             name: 'type',
                             label: 'Type',
                             items: [
-                                {value: 'trek', text: 'Trek'},
-                                {value: 'touristicContent', text: 'Touristic content'},
-                                {value: 'touristicEvent', text: 'Touristic event'},
-                                {value: 'outdoorSite', text: 'Outdoor site'},
+                                {value: 'trek', text: gettext('Trek')},
```

Comment:
Basculer ces 4 translation strings sur le système avec l'objet `window.i18n`.

On pourrait faire marcher le `gettext` en ajoutant une View `JavaScriptCatalog` qui piocherait dans les traductions métiers de Geotrek. Là on va partir sur traduire les valeurs du JS avec le `window.i18n`.

Reply:
Thanks for your review, it's also done :fire: